apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/ir/FrontendIrMergeUtils.kt

```diff
@@ -7,45 +7,6 @@ import com.apollographql.apollo.compiler.frontend.GQLSelectionSet
 import com.apollographql.apollo.compiler.frontend.responseName
 
 internal object FrontendIrMergeUtils {
-  /**
-   * Merges fragments that defined on the same type condition into one:
-   *
-   * ```
-   * query TestOperation {
-   *   random {
-   *       ... on Being {
-   *           name
-   *           friends {
-   *               name
-   *           }
-   *       }
-   *       ... on Wookie {
-   *          race
-   *          friends {
-   *            lifeExpectancy
-   *          }
-   *       }
-   *       ... on Being {
-   *           friends {
-   *              id
-   *           }
-   *       }
-   *   }
-   *}
-   * ```
-   *
-   * fragments defined on `Being` are going to be squashed into 1 while fragment `on Wookie` remains intact
-   */
-  fun List<GQLInlineFragment>.mergeInlineFragmentsWithSameTypeConditions(): List<GQLInlineFragment> {
-    return this
-        .groupBy { fragment -> fragment.typeCondition.name }
-        .map { (_, groupedFragments) ->
-          groupedFragments.drop(1).fold(groupedFragments.first()) { result, fragment ->
-            result.merge(fragment)
-          }
-        }
-  }
-
```

Comment:
why this was removed? I actually need it in my upcoming PR.

Reply:
That's done in the FrontendIr: https://github.com/apollographql/apollo-android/blob/152b0c7d7c6a38c71ca56e1e6ebe00a052ef7e7a/apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/frontend/ir/gql_to_frontend_ir.kt#L161